view/frontend/web/css/source/_extend.less

```diff
@@ -274,6 +274,21 @@
     }
 }
 
+#payment-method-braintree-lpm {
+    .primary {
+        width: 100%;
+
+        button.action {
+            margin-bottom: 1rem;
```

Comment:
Do you need this to be the same regardless of site? Or you want it to be affected by root font size

If a site has a root font size of say 18px this will appear bigger

Reply:
Honestly, this was pieced together from other CSS used in the module. There is no FE assigned to this, so I've had to make do.

What do you recommend?

Reply:
Personally I'd only use `rem` when I want the value to be affected by root size so typically typographic elements. I think you could just use pixels here. If it's possible to reduce the specificity too that'd be awesome (so just `.action`)